projects/packages/waf/src/class-waf-standalone-bootstrap.php

```diff
@@ -128,15 +128,19 @@ public function generate() {
 			throw new Exception( 'Can not work without the file system being initialized.' );
 		}
 
-		$bootstrap_file    = $this->get_bootstrap_file_path();
-		$mode_option       = get_option( Waf_Runner::MODE_OPTION_NAME, false );
-		$share_data_option = get_option( Waf_Runner::SHARE_DATA_OPTION_NAME, false );
+		$bootstrap_file                 = $this->get_bootstrap_file_path();
+		$mode_option                    = get_option( Waf_Runner::MODE_OPTION_NAME, false );
+		$automatic_rules_enabled_option = get_option( Waf_Runner::AUTOMATIC_RULES_ENABLED_OPTION_NAME, false );
+		$ip_lists_enabled_option        = get_option( Waf_Runner::IP_LISTS_ENABLED_OPTION_NAME, false );
+		$share_data_option              = get_option( Waf_Runner::SHARE_DATA_OPTION_NAME, false );
 
 		// phpcs:disable WordPress.PHP.DevelopmentFunctions
 		$code = "<?php\n"
 			. sprintf( "define( 'DISABLE_JETPACK_WAF', %s );\n", var_export( defined( 'DISABLE_JETPACK_WAF' ) && DISABLE_JETPACK_WAF, true ) )
 			. "if ( defined( 'DISABLE_JETPACK_WAF' ) && DISABLE_JETPACK_WAF ) return;\n"
 			. sprintf( "define( 'JETPACK_WAF_MODE', %s );\n", var_export( $mode_option ? $mode_option : 'silent', true ) )
+			. sprintf( "define( 'JETPACK_WAF_AUTOMATIC_RULES_ENABLED', %s );\n", var_export( $automatic_rules_enabled_option, true ) )
+			. sprintf( "define( 'JETPACK_WAF_IP_LISTS_ENABLED', %s );\n", var_export( $ip_lists_enabled_option, true ) )
```

Comment:
Also including the IP lists enabled constant here so we can check if any rules are enabled in `Waf_Runner::initialize()` before running the WAF.

Reply:
Removed